scraping/scraping/spiders/asos.py

```diff
@@ -72,20 +63,21 @@ def parse_SERP(self, response: ScrapyHttpResponse) -> Iterator[ScrapyHttpRequest
             yield ScrapyHttpRequest(
                 url=f"{self._product_api}{product_id}{self._filters}",
                 callback=self.parse_PRODUCT,
-                meta={"original_URL": response.url},
-                priority=1,  # higher prio than SERP => finish product requests first
+                meta=self.create_default_request_meta(response),
+                priority=2,  # higher prio than SERP => finish product requests first
             )
 
         # Pagination: Request SERPS if we are on start_url (offset=0)
         if "offset=0" in response.url:
-            yield from self.request_SERPs(url=response.url, data=data)
+            yield from self.request_SERPs(url=response.url, data=data, meta=response.meta)
```

Comment:
As discussed offline, since we have multiple different categories per spider the category information is no more globally stored and we make use of the request `meta` attribute to propagate the category information from the start_request to the SERP_requests and finally to the product_requests.